linuxNew/jdk/redhat/src/main/packaging/Dockerfile

```diff
@@ -1,25 +1,24 @@
 FROM fedora:33
 
-RUN dnf update -y && dnf install -y rpmdevtools rpm-sign rpmlint gnupg2 wget tar dpkg findutils
+RUN dnf update -y && dnf install -y rpmdevtools rpm-sign rpmlint \
```

Comment:
Any particular reason why the first 3 are on one line then each of the others are on their own? Seems inconsistent.

Reply:
do not want to keep the line too long.
can make each package one line instead